Resources/Prototypes/Nyanotrasen/Entities/Clothing/psionic_clothing.yml

```diff
@@ -64,3 +64,18 @@
   - type: ClothingGrantPsionicPower
     power: NoosphericZapPower
   - type: Psionic
+
+- type: entity
+  parent: ClothingNeckBase
+  id: RecoveryConePsionic
+  name: radar dish
+  description: yo, is shark week on yet?
+  components:
+  - type: IngestionBlocker
+  - type: Sprite
+    sprite: Nyanotrasen/Clothing/Neck/felinidcone.rsi
+  - type: Clothing
+    sprite: Nyanotrasen/Clothing/Neck/felinidcone.rsi
+  - type: ClothingGrantPsionicPower
+    power: TelegnosisPower
```

Comment:
We had an RF sensitivity power at some point. That seems most fitting, if it ever comes back. Otherwise we already have glasses that grant telegnosis. Just remove this item...?

Reply:
since it is a radar dish, i think psionic pulse might be more fitting